openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/LightGBMUtils.java

```diff
@@ -82,6 +84,31 @@ static CpuArchitecture getCpuArchitecture(final String cpuArchName) {
         }
     }
 
+    /**
+     * Checks if the current operative system is Alpine.
+     *
+     * @return true when the operative system is Alpine and false otherwise.
+     */
+    static private boolean isAlpine() {
+        final String osReleasePath = "/etc/os-release";
+
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(osReleasePath));
```

Comment:
Can you move this to a try-with-resources?

Reply:
This code was removed in order to use an alternative detection of a mus-based OS using an env variable.